src/Model/Config/NostoConfigService.php

```diff
@@ -78,6 +78,8 @@ class NostoConfigService
 
     public const ENABLE_PRODUCT_LABELLING_SYNC = 'enableLabelling';
 
+    public const ENABLE_STORE_DATA_ABUNDANT_CARTS = 'storeDataAbundantCarts';
```

Comment:
this would be probably storeAbandonedCartData (abandoned means as forgotten while abundant means as "a lot") :)

Reply:
Obviously means it also would be ENABLE_STORE_ABANDONED_CART_DATA